wandb/integration/magic.py

```diff
@@ -294,6 +294,10 @@ def _magic_fit(
     *args,
     **kwargs
 ):
+    if hasattr(self, "_wandb_internal_model"):
+        return self._fit(
+            x=x, y=y, batch_size=batch_size, epochs=epochs, *args, **kwargs
+        )
```

Comment:
When would this code get executed?  We should definitely add a test to `test_keras` that assers we actually log gradients as well.

Reply:
This code is reached when `.fit()` is called on the grad accumulator model (if in magic mode). I have left a comment on the `_wandb_internal_model` attribute in `keras.py`